packages/reports/src/Constants/__tests__/data-table-constants.spec.tsx

```diff
@@ -0,0 +1,570 @@
+    });
+    it('should render cell content for "Balance" column', () => {
+        const balanceColumn = columns[6];
+        const cell_value = '2000';
+        //@ts-expect-error type for cell_value is wrong
+        render(balanceColumn.renderCellContent({ cell_value }));
+
+        expect(screen.getByText(cell_value)).toBeInTheDocument();
+    });
+});
+
+describe('getProfitTableColumnsTemplate', () => {
+    const currency = 'USD';
+    const items_count = 10;
+    const columns = getProfitTableColumnsTemplate(currency, items_count);
+
+    it('should render cell content for "Type" column', () => {
+        const typeColumn = columns[0];
+        const row_obj = { transaction_id: '123' };
+        //@ts-expect-error renderCellContent conditionally renders null or JSX ELement hence render function complains
+        render(typeColumn.renderCellContent({ row_obj, is_footer: false }));
+
+        expect(screen.getByText(row_obj.transaction_id)).toBeInTheDocument();
+    });
+    it('should render footer content for "Type" column', () => {
+        const typeColumn = columns[0];
+        //@ts-expect-error renderCellContent conditionally renders null or JSX ELement hence render function complains
+        render(typeColumn.renderCellContent({ is_footer: true }));
+
+        expect(screen.getByText(/on the last/i)).toBeInTheDocument();
+    });
+    it('should render cell content for "Currency" column', () => {
+        const currencyColumn = columns[2];
+        //@ts-expect-error renderCellContent conditionally renders null or JSX ELement hence render function complains
+        render(currencyColumn.renderCellContent({ is_footer: false }));
+
+        expect(screen.getByText(currency)).toBeInTheDocument();
+    });
+    it('should render cell content for "Buy time" column', () => {
+        const buyTimeColumn = columns[3];
+        const cell_value = '2023-05-30 12:34:56';
+        //@ts-expect-error renderCellContent conditionally renders null or JSX ELement hence render function complains
+        render(buyTimeColumn.renderCellContent({ cell_value, is_footer: false }));
+
+        expect(screen.getByText(`${cell_value} GMT`)).toBeInTheDocument();
+    });
+    it('should render cell content for "Stake" column', () => {
+        const stakeColumn = columns[4];
+        const cell_value = '1000';
+        //@ts-expect-error renderCellContent conditionally renders null or JSX ELement hence render function complains
+        render(stakeColumn.renderCellContent({ cell_value, is_footer: false }));
+
+        expect(screen.getByText(cell_value)).toBeInTheDocument();
+    });
+    it('should render cell content for "Sell time" column', () => {
+        const sellTimeColumn = columns[5];
+        const cell_value = '2023-05-30 13:34:56';
+        //@ts-expect-error renderCellContent conditionally renders null or JSX ELement hence render function complains
+        render(sellTimeColumn.renderCellContent({ cell_value, is_footer: false }));
```

Comment:
```suggestion
        render(sellTimeColumn.renderCellContent({ cell_value, is_footer: false }) as JSX.Element);
```
will this work instead of the //@ts-expect-error comment? :)